### PR TITLE
write app version

### PR DESCRIPTION
**Pull request by furkhat**

### CODE REVIEW

File: tool/dappinst/main.go [hunk 1]
```diff
@@ -133,3 +135,15 @@ func createDappvpnConfig(logger *util.Logger,
 		logger.Fatal("failed to write dappvpn config")
 	}
 }
+
+func writeAppVersion(logger *util.Logger, db *reform.DB) {
```
Comment (sofabeat) [inline tool/dappinst/main.go]: Let's write to log:
1. Old version (before update)
2. New version (after update)

Comment (furkhat) [reply tool/dappinst/main.go]: how I get version before update?

Comment (furkhat) [reply tool/dappinst/main.go]: Is this script meant to be run on non-clean db?

Comment (sofabeat) [reply tool/dappinst/main.go]: This script runs on non-clean DB. Main goal of this this script to set unique authentication parameters both in DB and config files of corresponding service adapters (currently dappvpn).

**Pull request merged**